src/test/java/org/kiwiproject/dropwizard/error/ErrorContextBuilderTest.java

```diff
@@ -130,6 +146,53 @@ private void assertIllegalArgumentExceptionThrownBuilding(SoftAssertions softly,
         }
     }
 
+    @Nested
+    class CleanupJob {
+
+        @Test
+        void shouldHaveCleanupApplicationErrorsJob() {
+            var cleanupConfig = new CleanupConfig();
+            cleanupConfig.setCleanupJobName("foo-%d");
+
+            var builder = ErrorContextBuilder.newInstance()
+                    .environment(environment)
+                    .serviceDetails(serviceDetails)
+                    .cleanupConfig(cleanupConfig)
+                    .dataStoreType(DataStoreType.NOT_SHARED);
+
+            builder.buildInMemoryH2();
+
+            var dataSourceFactory = ApplicationErrorJdbc.createInMemoryH2Database();
+            builder.buildWithDataStoreFactory(dataSourceFactory);
+
+            var jdbi = Jdbi3Builders.buildManagedJdbi(environment, dataSourceFactory);
+             builder.buildWithJdbi3(jdbi);
+
+             verify(executor, times(3))
+                     .scheduleWithFixedDelay(any(CleanupApplicationErrorsJob.class), eq(1L),
```

Comment:
Instead of hard coding the expected values, we should verify against the values in the `CleanupConfig`. And perhaps we should change _all_ the default values in `CleanupConfig` so we can verify that all of them are correctly used when scheduling.

Also, earlier in the test we `cleanupConfig.setCleanupJobName("foo-%d")` but we never verify that. We should verify the exact call to `scheduledExecutorService` (which I mentioned in an earlier comment re: daemon threads), e.g.

```java
verify(environment.lifecycle()).scheduledExecutorService("foo-%d", true);
```